nbs/styles.css

```diff
@@ -35,3 +35,7 @@ div.description {
   font-size: 135%;
```

Comment:
_Commented on notebook `nbs/styles.css` Cell 1 Line 39_
```css
  padding-left: 2px;
  padding-top: 5px;
  font-style: italic;
  font-size: 135%;
  opacity: 70%;
}

img {
```
test

_Respond and view the context [here](http://localhost:3000/GitNotebooks/docs/pull/2)._